commcare_connect/opportunity/tables.py

```diff
@@ -952,7 +952,10 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault("order_by", "user__name")
         super().__init__(*args, **kwargs)
 
-    def render(self, value):
+    def render(self, value, record):
+        if not record.accepted:
+            return "-"
```

Comment:
If it's None, wouldn't it use default empty label. We don't need to return `-` right?